### PR TITLE
Update branch metadata

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Review by ostrolucky — APPROVED** [review]

**Comment by greg0ire** [comment]
I should have waited for the build before requesting approvals, sorry :sweat_smile:

**Comment by greg0ire** [comment]
Blocked by https://github.com/doctrine/coding-standard/pull/226 I suppose

**Pull request merged**
